### PR TITLE
fix deprecation warning for timeout

### LINKED ISSUE #5: Timeout.timeout
The following output is being seen when ran with the current embedded ruby:
> /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-ftp-1.1.0/bin/check-ftp.rb:89:in `run': Object#timeout is deprecated, use Timeout.timeout instead.

[Line 89](https://github.com/sensu-plugins/sensu-plugins-ftp/blob/ea746ee200f78085552ff7338ec0dc37f499529e/bin/check-ftp.rb#L89) should be changed from : `timeout(config[:timeout]) do`

to: `Timeout.timeout(config[:timeout]) do`

Expected output:

> CheckFTP OK

### PR DESCRIPTION
**Pull request by majormoses**
## Pull Request Checklist

fixes #5 

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose
remove deprecation warning

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 #### General
 
-- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
+- [ ] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
```
Comment (eheydrick) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: extra space after Guidelines.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
+This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
```
Comment (eheydrick) [inline CHANGELOG.md]: ditto

**Review by eheydrick — COMMENTED** [review]
Fix looks good. More nitpicky formatting on the markdown.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (majormoses) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I clearly did not copy/paste...

**Comment by majormoses** [comment]
@eheydrick fixed.

**Review by eheydrick — APPROVED** [review]

**Pull request merged**